packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue

```diff
@@ -1475,8 +1475,6 @@
                             _method: element.getAttribute("data-method")
                         })
                         .then(function(response) {
-                            this.result = response;
```

Comment:
The response should be passed to `this. result`  props. Because other methods will use this.